pkg/claims/installation.go

```diff
@@ -158,18 +168,70 @@ func (i *Installation) SetLabel(key string, value string) {
 	i.Labels[key] = value
 }
 
+// EncodeSensitiveParameter encodes parameters that contains sensitive data so
+// it can be associated with the current installation record.
+func (i Installation) EncodeSensitiveParameter(param secrets.Strategy) secrets.Strategy {
```

Comment:
nit: Let's try to come up with a more descriptive name for this function. Encode implies that we are changing the data, like HTML encoding, URL encoding, etc. As I understand it, this function is making the sensitive parameter unique to the installation when saved into the secret store right?